web/satellite/src/utils/converter.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+const KB = 1e3;
+const MB = 1e6;
+const GB = 1e9;
+const TB = 1e12;
+const PB = 1e15;
+
+/**
+ * Used to display correct and convenient data
+ */
+export class Converter {
+    /**
+     * Used to format amount from bytes to more compact unit
+     * @param bytes - holds amount of bytes
+     * @returns bytes - amount of formatted bytes
+     */
+    public static formatBytes(bytes: number): string {
```

Comment:
its better to have different methods, like Size.GiB(), Size.Mib()
having switch here breaks single responsibility principle